Source/WebCore/rendering/RenderLayerBacking.cpp

```diff
@@ -4075,12 +4081,10 @@ bool RenderLayerBacking::updateAcceleratedEffectsAndBaseValues()
                 continue;
             if (animatesWidth || animatesHeight) {
                 auto& blendingKeyframes = effect->blendingKeyframes();
-                if (blendingKeyframes.hasWidthDependentTransform() && animatesWidth)
-                    continue;
-                if (blendingKeyframes.hasHeightDependentTransform() && animatesHeight)
-                    continue;
+                if ((blendingKeyframes.hasWidthDependentTransform() && animatesWidth) || (blendingKeyframes.hasHeightDependentTransform() && animatesHeight))
+                    disallowedAcceleratedProperties.add(transformRelatedAcceleratedProperties);
```

Comment:
I liked it the original way.

But also, put the already calculated value before the call to the function so it can early return if necessary.